### PR TITLE
fix(browse): search bar function implemented

### LINKED ISSUE #225: Search bar in `/browse` page does not seem to work
### Description

Currently, the search bar in the `/browse` page seems only to be an input field with no search function implemented.

#### How to reproduce

Run the current UI project and login to the `/browse` page and try to use the search bar above the table.

#### Screenshots

Screenshot from 2022-06-16 12-04-40

### Versions

* Last commit id on master:3003944
* Operating System (lsb_release -a):Ubuntu 20.04 LTS
* Browser [e.g. chrome, safari]: Firefox, Chrome

### PR DESCRIPTION
**Pull request by soham4abc**
## Description

Search function implemented in the `/browse` page

### Changes

`pages/Browse/index.jsx`

## How to test

Run the UI project with the backend and try to search the uploaded files in the `/browse` page

## Video
Screencast from 19-06-22 12 05 02 PM IST

Fixes #225

### CODE REVIEW

File: src/pages/Browse/index.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /*
- Copyright (C) 2021 Shruti Agarwal (mail2shruti.ag@gmail.com), Aman Dwivedi (aman.dwivedi5@gmail.com)
+ Copyright (C) 2021 Shruti Agarwal (mail2shruti.ag@gmail.com), Aman Dwivedi (aman.dwivedi5@gmail.com),
+ Soham Banerjee (sohambanerjee4abc@hotmail.com)
```
Comment (GMishx) [inline src/pages/Browse/index.jsx:3]: Please add copyright year.

File: src/pages/Browse/index.jsx [hunk 2]
```diff
@@ -234,7 +237,8 @@ const Browse = () => {
                     <input
                       type="search"
                       className="form-control"
-                      placeholder="Search"
+                      placeholder="Search in Page"
```
Comment (GMishx) [inline src/pages/Browse/index.jsx:240]: Better name will be "Filter upload" IMO.

**Review by GMishx — COMMENTED** [review]
Does the filter for "Status" and "Assignee" working?

For me, it makes no difference.

**Comment by soham4abc** [comment]
> Does the filter for "Status" and "Assignee" working?
> 
> For me, it makes no difference.

No sir this PR was only focused on the search bar.

File: src/pages/Browse/index.jsx (hunk 2 above)
Comment (soham4abc) [reply src/pages/Browse/index.jsx:240]: Resolved

**Review by GMishx — APPROVED** [review]
Changes looks good. Working as expected.

**Pull request merged**